cnxeasybake/oven.py

```diff
@@ -269,6 +270,13 @@ def bake(self, element, last_step=None):
 
         # Do label/link updates
 
+        # Add an empty string to each element just to make sure the element is closed
```

Comment:
I opted to put it in here so that the tests would generate the expected output and if someone included easybake as a library it would also generate the expected output. 

What would be the reasons for including it in the cli script? Is it that it's a "serialization" choice rather than a core part of baking? (I don't have a reason except that it would result in copy-pasta code in the CLI as well as the unit tests)

I'll update the comment but we can keep chatting 😄